src/snowflake/connector/auth/_auth.py

```diff
@@ -166,10 +168,14 @@ def authenticate(
         mfa_callback: Callable[[], None] | None = None,
         password_callback: Callable[[], str] | None = None,
         session_parameters: dict[Any, Any] | None = None,
-        timeout: int = 120,
+        # max time waiting for MFA response, currently unused
```

Comment:
this comment is a bit confusing, can you point me to where it's used in the MFA reponse?